jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -136,6 +138,214 @@ ecma_builtin_typedarray_prototype_length_getter (ecma_value_t this_arg) /**< thi
 } /* ecma_builtin_typedarray_prototype_length_getter */
 
 /**
+ * Type of routine.
+ */
+typedef enum
+{
+  TYPEDARRAY_ROUTINE_EVERY, /**< routine: every */
```

Comment:
Bit nitpicky, but it would great to write the section numbers of the ecma standard to each routine.